.github/workflows/ai-pr-reviewer.yml

```diff
@@ -0,0 +1,30 @@
+name: Code Review
+
+permissions:
+  contents: read
+  pull-requests: write
+
+on:
+  pull_request:
+  pull_request_review_comment:
+    types: [created]
+
+concurrency:
+  group:
+    ${{ github.repository }}-${{ github.event.number || github.head_ref ||
+    github.sha }}-${{ github.workflow }}-${{ github.event_name ==
+    'pull_request_review_comment' && 'pr_comment' || 'pr' }}
+  cancel-in-progress: ${{ github.event_name != 'pull_request_review_comment' }}
+
+jobs:
+  review:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: coderabbitai/ai-pr-reviewer@latest
+        env:
+          GITHUB_TOKEN: ${{ secrets.AI_PR_GITHUB_TOKEN }}
+          OPENAI_API_KEY: ${{ secrets.AI_PR_OPENAI_API_KEY }}
+        with:
+          debug: false
+          review_simple_changes: false
+          review_comment_lgtm: false
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The workflow file seems to be well structured and follows the standard GitHub Actions format. However, there are a few points that need attention:

1. The `permissions` section is set to read contents and write pull requests. This is good for security as it limits the scope of what this action can do.

2. The `concurrency` section is used to ensure that only one instance of this workflow runs at a time per pull request. This is a good practice to prevent race conditions.

3. The `jobs` section specifies that this workflow will run on the latest version of Ubuntu and uses the `coderabbitai/ai-pr-reviewer@latest`. It's generally recommended to pin actions to a specific commit or version for stability and security reasons. Using `latest` could potentially introduce breaking changes or vulnerabilities if the action is updated.

4. The environment variables `GITHUB_TOKEN` and `OPENAI_API_KEY` are fetched from secrets which is a secure way to handle sensitive data.

5. In the `with` section, `debug`, `review_simple_changes`, and `review_comment_lgtm` are all set to false. Depending on your needs, you might want to enable some of these options. For example, setting `review_simple_changes` to true would make the bot review even simple changes, which could be useful for catching small mistakes.

Here's a suggested change to pin the action to a specific version:

```diff
-       - uses: coderabbitai/ai-pr-reviewer@latest
+       - uses: coderabbitai/ai-pr-reviewer@v1.0.0
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->